PendingReleaseNotes

```diff
@@ -77,3 +77,28 @@
   librbd to load the plugin by adding the following to your configuration::
 
     rbd_plugins = parent_cache
+
+* `blacklist` has been replaced with `blocklist` throughout.  The following commands have changed:
+
+  - ``ceph osd blacklist ...`` are now ``ceph osd blocklist ...``
```

Comment:
```suggestion
  - ``ceph osd blacklist ...`` is now ``ceph osd blocklist ...``
```

Reply:
I did 'are' because the '...' refers to a bunch of commands.